follow_fork/README.md

```diff
@@ -0,0 +1,27 @@
+Follow 'fork()'
+=============
```

Comment:
This line should be as long as the title.
It's backticks (`) not apostrophe.

Reply:
fixed

Reply:
I don't think you fixed this:
> This line should be as long as the title.

Reply:
oops, bad glasses!